src/main/java/com/example/OptionalTask_2.java

```diff
@@ -0,0 +1,33 @@
+//2.Вывести числа в порядке возрастания (убывания) значений их длины.
+
+package com.example;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class OptionalTask_2 {
+    public static void main(String[] args) {
+        int[] integers = readLine();
+        Arrays.sort(integers);
+        for (int element : integers) {
+            System.out.printf(element + " ");
```

Comment:
No need in `printf` if you don't use simple concatenation and don't use additional arguments.